frontend/region-report.js

```diff
@@ -278,87 +278,50 @@ let getClericalUnrestBlock = function(r) {
 			crisisDescription += "\nGoal: " + r.name + " has no neighboring enemy regions.";
-			crisisDescription += cd("Inspiring", "Untrusting");
 		} else if (r.noble.crisis.type == "ENNUI") {
 			crisisDescription = r.noble.name + " is bored with life."
 			crisisDescription += "\nGoal: " + r.kingdom + " throws a feast in the region.";
-			crisisDescription += cd("Generous", "Workaholic");
 		} else if (r.noble.crisis.type == "CULTISM") {
 			crisisDescription = r.noble.name + " is being woo'd by the Cult of the Witness."
 			crisisDescription += "\nGoal: A new temple is built in " + r.name + ".";
-			crisisDescription += cd("Pious", "Cultist");
 		} else if (r.noble.crisis.type == "OVERWHELMED") {
 			crisisDescription = r.noble.name + " needs to be shown how to better govern the region."
 			crisisDescription += "\nGoal: A character loyal to " + r.kingdom + " performs the Govern action in " + r.name + ".";
-			crisisDescription += cd("Meticulous", "Wasteful");
 		} else if (r.noble.crisis.type == "UPRISING") {
 			crisisDescription = r.noble.name + " is facing a popular rebellion."
 			crisisDescription += "\nGoal: Popular unrest in " + r.name + " is 50% or less.";
-			crisisDescription += cd("Soothing", "Tyrannical");
 		} else if (r.noble.crisis.type == "STARVATION") {
 			crisisDescription = r.noble.name + " is starving alongside their people."
 			crisisDescription += "\nGoal: Starvation in " + r.name + " ceases.";
-			crisisDescription += cd("Rationing", "Desperate");
 		} else if (r.noble.crisis.type == "GUILD") {
 			crisisDescription = r.noble.name + " is losing power to the local guilds."
 			crisisDescription += "\nGoal: A new construction is built in " + r.name + ".";
-			crisisDescription += cd("Patronizing", "Broke");
 		}
 		crisisDescription += "\nDue at the end of week " + r.noble.crisis.deadline + ".";
-		crisis = `<tooltip-element tooltip="All nobles experience crises every six weeks. If resolved positively within six weeks, a crisis gives a positive trait to the noble and decreases noble unrest by 25 percentage points. If unresolved within the deadline, the noble gains a negative trait and 12 percentage points of unrest.">Crisis:</tooltip-element><tooltip-element id="crisis" tooltip="${crisisDescription}">${capitalizeForce(r.noble.crisis.type)}</tooltip-element>`
+		crisis = `<tooltip-element tooltip="All nobles experience crises every six weeks. If resolved positively within six weeks, a crisis decreases noble unrest by 25 percentage points. If unresolved within the deadline, the noble gains 25 percentage points of unrest.">Crisis:</tooltip-element><tooltip-element id="crisis" tooltip="${crisisDescription}">${capitalizeForce(r.noble.crisis.type)}</tooltip-element>`
 	}
+	let levelDetail = "Nobles gain an experience point each turn, and have a level equal to the square root of one plus their experience. This noble's level gives the region:";
+	levelDetail += "\n +" + Math.round(r.calcNobleLevel() * 0.1 * 100) + "% regional tax";
```

Comment:
This would be a good place to use some String interpolation to get those numbers in there IMO.

Reply:
Done.